src/Compilers/CSharp/Test/Semantic/FlowAnalysis/LocalFunctions.cs

```diff
@@ -1550,5 +1550,49 @@ System.Collections.Generic.IEnumerable<string> getGoo(int count, out bool output
                 //         System.Collections.Generic.IEnumerable<string> getGoo(int count, out bool goobar)
                 Diagnostic(ErrorCode.ERR_ParamUnassigned, "getGoo").WithArguments("output").WithLocation(6, 56));
         }
+
+        [Fact]
+        [WorkItem(41631, "https://github.com/dotnet/roslyn/issues/41631")]
+        public void UseOfCapturedVariableAssignedByOutParameter()
+        {
+            CreateCompilation(@"
+public class C
+{
+    void M()
+    {
+        string s0;
+        local1(out s0);
+    
+        void local1(out string s1)
+        {
+            s0.ToString(); // should produce an error, but doesn't
```

Comment:
please remove the comment :)

Reply:
I haven't seen `// 1` before. What's it for?

Reply:
it's a convention that makes it easier to tell at a glance which lines are expected to produce diagnostics. So if your test produced several diagnostics you may choose to annotate some lines with `// 1`, `// 2`, etc.